elytron/src/main/java/org/wildfly/extension/elytron/DistributedRealmDefinition.java

```diff
@@ -101,21 +120,32 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
             final List<InjectedValue<SecurityRealm>> distributedRealmValues = new ArrayList<>();
 
+            boolean ignoreUnavailableRealms = IGNORE_UNAVAILABLE_REALMS.resolveModelAttribute(context, model).asBoolean();
+            boolean emitEvents = EMIT_EVENTS.resolveModelAttribute(context, model).asBoolean();
+
+            List<String> distributedRealms = REALMS.unwrap(context, model);
+
             TrivialService<SecurityRealm> distributedRealmService = new TrivialService<SecurityRealm>(() ->
             {
                 SecurityRealm[] realms = new SecurityRealm[distributedRealmValues.size()];
 
+                Consumer<Integer> unavailableRealmConsumer = emitEvents ? (realmIndex) -> {
```

Comment:
@lvydra jsut a minor, should here be 

```
emitEvents && ignoreUnavailableRealms ? (realmIndex) -> ...
```

so that the emitting callback is registered only when the unavailable realm should be ignored?

Reply:
@Skyllarr logic on Elytron side is taking care of that, however, it seems better to add it here too.